state/dashboard/__tests__/selectors.test.js

```diff
@@ -0,0 +1,70 @@
+import {
+  isDashboardMoviesLoading,
+  dashboardMoviesList,
+  dashboardMoviesTotalPages,
+  dashboardMoviesCurrentPage,
+  dashboardMoviesError,
+  dashboardMoviesSearchQuery
+} from '../selectors';
+
+describe('Dashboard selectors.', () => {
+  const loading = false;
+  const error = 'Mock error';
+  const totalItems = 21;
+  const currentPage = 2;
+  const searchQuery = 'Mock query';
+  const state = {
+    dashboard: {
+      loading,
+      error,
+      ids: [1, 2],
+      totalItems,
+      currentPage,
+      searchQuery
+    },
+    data: {
+      movies: {
+        1: { id: 1, title: 'First mock title' },
+        2: { id: 2, title: 'Second mock title' }
+      }
+    }
+  };
+
+  it('isDashboardMoviesLoading() returns loading status', () => {
+    expect(isDashboardMoviesLoading(state)).toEqual(loading);
```

Comment:
You can use .toBeFalsy()
https://jestjs.io/docs/en/expect#tobefalsy

Reply:
Fixed